deepspeed/module_inject/policy.py

```diff
@@ -0,0 +1,108 @@
+'''
+Copyright 2022 The Microsoft DeepSpeed Team
+'''
+from abc import ABC
+from deepspeed.utils.types import ActivationFuncType
+
+
+class DSPolicy(ABC):
+    _orig_layer_class = None
+
+    def __init__(self):
+        self.cuda_graph_supported = False
+
+    def attention(self):
```

Comment:
```suggestion
    @abstractmethod
    def attention(self):
```
Using `abstractmethod` will provide an explicit `Exception` at the time of instantiation for inheriting classes that do no implement this method.